package.json

```diff
@@ -28,16 +29,19 @@
         "shell": "ts-node --project packages/matter-node-shell.js/tsconfig.dist.json packages/matter-node-shell.js/src/app.ts"
     },
     "devDependencies": {
-        "@typescript-eslint/eslint-plugin": "^5.62.0",
-        "@typescript-eslint/parser": "^5.62.0",
-        "eslint": "^8.47.0",
+        "@typescript-eslint/eslint-plugin": "^6.6.0",
+        "@typescript-eslint/parser": "^6.6.0",
+        "eslint": "^8.48.0",
         "eslint-import-resolver-typescript": "^3.6.0",
         "eslint-plugin-import": "^2.28.1",
         "prettier": "^3.0.2",
         "prettier-plugin-organize-imports": "^3.2.3",
-        "typedoc": "^0.24.8",
+        "typedoc": "^0.25.1",
         "typedoc-plugin-markdown": "^3.15.4",
-        "typedoc-plugin-missing-exports": "^2.0.1",
-        "typescript": "^4.9.5"
+        "typedoc-plugin-missing-exports": "^2.1.0",
+        "typescript": "^5.2.2"
+    },
+    "dependencies": {
+        "v8-profiler-next": "^1.9.0"
```

Comment:
as base dependency?

Reply:
Yeah, it's the same as the other dependencies (mocha etc.), it's used by the code published by the package...  `npm run test -- --profile` will dump a .cpuinfo file into build/profiles

Reply:
but mocha is a dev-Dependency and not an official one, but most likely ok fpr the base package to have ... ok

Reply:
Ah, sorry...  I didn't look at which tsconfig.json this was.  It should have only been in the tools/tsconfig.json, will remove here.